manifests/crc/default-db-included-installation/openshift/kruize-crc-openshift.yaml

```diff
@@ -93,6 +93,7 @@ data:
       "dbdriver": "jdbc:postgresql://",
       "plots": "false",
       "local": "false",
+      "logAllHttpReqAndResp": "true",
```

Comment:
This needs to be in all 4 yaml's, I only see two of them right now

Reply:
updated now